docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>parameter array <code>c</code> by <s>using</s> representing it </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Good catch!</p>